generate_settings.py

```diff
@@ -92,6 +92,8 @@ def arch2arch(arch):
         return "rv64i"
     if arch == "rv64ima":
         return "rv64im"
+    if arch == "rv64imafdcv":
+        return "rv64imafdcv_zfh"
```

Comment:
I assume this is because the hardware generator is missing the zfh extension. I'd think the Correct place to fix this is in the hardware generator, but if we need this here for now, I'd like to have a comment explaining why.

Reply:
> I assume this is because the hardware generator is missing the zfh extension. I'd think the Correct place to fix this is in the hardware generator, but if we need this here for now, I'd like to have a comment explaining why.

Agree, will add that